examples/multicsc/plot_sample_evoked_response.py

```diff
@@ -76,41 +85,74 @@
 
 
 ###############################################################################
-# Here, we load the data from the sample datase. The data is not epoched yet,
-# but we split it into 12 parts to make the most of multiple processors during
-# the model fitting.
+# Load the sample data from MNE-python and select the gradiometer channels.
+# The MNE sample data contains MEG recordings of a subject with visual and
+# auditory stimuli. We load the data using utilities from MNE-python as a Raw
+# object and select the gradiometers from the signal.
+
+import os
+import mne
+import numpy as np
+
+print("Loading the data...", end='', flush=True)
+data_path = mne.datasets.sample.data_path()
+subjects_dir = os.path.join(data_path, "subjects")
+data_dir = os.path.join(data_path, 'MEG', 'sample')
+file_name = os.path.join(data_dir, 'sample_audvis_raw.fif')
+raw = mne.io.read_raw_fif(file_name, preload=True, verbose=False)
+raw.pick_types(meg='grad', eeg=False, eog=False, stim=True)
+print('done')
+
+
+###############################################################################
+# Then, we remove the powerline artifacts and high-pass filter to remove the
+# drift which can impact the CSC technique. The signal is also resampled to
+# 150 Hz to reduce the computationnal burden.
+
+print("Preprocessing the data...", end='', flush=True)
+raw.notch_filter(np.arange(60, 181, 60), n_jobs=n_jobs, verbose=False)
+raw.filter(2, None, n_jobs=n_jobs, verbose=False)
+raw = raw.resample(sfreq, npad='auto', n_jobs=n_jobs, verbose=False)
+print('done')
+
+
+###############################################################################
+# Load the data as an array and split it in chunks to allow parallel processing
+# during the model fit. Each split is considered as independent.
+# To reduce the impact of border artifacts, we use `apply_window=True`
+# which scales down the border of each split with a tukey window.
+
+from alphacsc.utils.signal import split_signal
+X = raw.get_data(picks=['meg'])
+info = raw.copy().pick_types(meg=True).info  # info of the loaded channels
+X_split = split_signal(X, n_splits=n_splits, apply_window=True)
```

Comment:
should this be documented in the API page?

Reply:
Yes, this would be nice

Reply:
great, made a PR: https://github.com/alphacsc/alphacsc/pull/24